src/status_im2/contexts/syncing/how_to_pair/style.cljs

```diff
@@ -0,0 +1,101 @@
+(ns status-im2.contexts.syncing.how-to-pair.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def scroll-view
+  {:flex             1
+   :background-color colors/neutral-95})
+
+(def container-outer
+  {:style {:background-color colors/neutral-95
+           :justify-content  :space-between
+           :align-items      :left
+           :padding          16
+           :border-radius    10}})
+
+(def container-instructions
+  {:flex               1
+   :padding-top        20
+   :padding-bottom     8
+   :padding-horizontal 16
+   :background-color   colors/neutral-95})
+
+(def heading
+  {:size   :heading-1
+   :weight :semi-bold
```

Comment:
Component props like `:size` and `:weight` should be in the view file, here there should be only styles definitions (check #15686 although is not merged)

Reply:
Correct definition should be like the one above, `container-instructions`, please check that all the styles in this file  follow the same pattern

Reply:
Thanks @briansztamfater! I moved the component props to the view file and refactored the view to reduce duplication of properties.